scripts/weekly-dep-report.sh

```diff
@@ -3,15 +3,12 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # fast fail.
-set -eo pipefail
-
-echo " Dependency change report "
-echo "--------------------------"
```

Comment:
maybe move this down to where the report is generated instead of removing it?

Reply:
The first line starts with `Dependency change in <>` anyways. So, felt this line makes the report unnecessarily bloated.  `set -xv` is the accurate way to debug I think.